kuma/static/js/contribution-handler.js

```diff
@@ -567,8 +575,9 @@
     customAmountInput.blur(function(event) {
         // Send GA Event.
         mdn.analytics.trackEvent({
-            category: 'Contribution popover',
-            action: 'Amount manually selected',
+            category: 'payments',
+            action: 'banner',
+            label: 'custom amount',
```

Comment:
For this event, the user's custom value is used as the event value. However, the user is allowed to enter a decimal, such as 2.4, and an analytics event value must be an integer. I think the value should be:

```
value: Math.max(0, Math.floor(event.target.value))
````

Reply:
Further playing with this, an event is fired if an invalid value is entered into the form, like ``foo``, which would result in the event ``NaN`` with my additions. I'm not sure if it is worth making the event handler bulletproof for any garbage input, but it would be fun.

Reply:
``value * 100`` works (US / UK only?), but then the radio selects should also be ``3200``, ``6400``.